docs/Troubleshooting.md

```diff
@@ -25,13 +25,85 @@ Brim invokes to extract flows from your pcap handles the most common variations
 we've encountered, but you may have come across a corner case that it wasn't
 prepared to handle.
 
-If the Brim app shows an error message, please [open a bug report](#opening-an-issue)
-and include a [screenshot](#screenshots-videos) that includes the error message
-or cut & paste the error message text into the issue.
-
-If you don't see an error message, or the error seems terse and unhelpful, it
-may help us get to the bottom of it faster if you include the debug info from
-[Developer Tools](#developer-tools).
+Such problems are often due to file format variations that are unique to the
+system that generated the capture. In such situations, the most expedient debug
+path would be to send us the capture file so our developers can use it to
+reproduce the problem, fix it, and create automated tests to ensure the problem
+doesn't arise again in the future. We understand that concerns regarding
+[sensitive information](http://0.0.0.0:6419/#sensitive-information-important)
```

Comment:
What's the intended destination for this link?

Reply:
@alfred-landrum: Gah! Thanks for catching that. It was supposed to be a link to an anchor within the same article. I just made a stupid mistake and pasted an absolute URL from when I was rendering the markdown locally with `grip`. I just pushed a commit that should fix the link.